string source_id    # å‘é€è€…çš„åå­—
string target_id    # æ¥æ”¶è€…çš„åå­— (å¦‚æœæ˜¯ "ALL" ä»£è¡¨ç¾¤å‘)
int32 generation      # ğŸ”¥ æ–°å¢ï¼šæ ‡è®°è¿™æ˜¯ç¬¬å‡ ä»£çš„ç§ç¾¤
Individual[] individuals
